Fibonacci/task.md

```diff
@@ -1,7 +1,20 @@
-Write a class called `Fibonacci` within the methode `calcFib (int n)` which returns
+Write a class called `Fibonacci` within the function `fibonacci (int n)` which returns
 the result of the Fibonacci function of given number `n`.
 
-Make sure that only legal arguments will be accepted otherwise throw an Exception `Fibonacci is not defined for neagtive Numbers!`.
+Make sure that only legal arguments will be accepted otherwise throw an
+IllegalArgumentException `Fibonacci is not defined for negative Numbers!`.
+
+## Example ##
+
+Here are some values to show, which results the function `fibonacci (n)` should return.
+
+`n` --> `fibonacci (n)`  
```

Comment:
I did it first, but INLOOP don't present it as as table. I think the [Leetspeak-task](https://github.com/st-tu-dresden/inloop-tasks/blob/master/LeetSpeek/task.md) has the same problem and solved it with a HTML table, but the default spacing between the columns is too close, in my opinion. So the arrows seemed to be a fair and clean way.